src/core/lv_obj_pos.c

```diff
@@ -783,15 +788,12 @@ void lv_obj_get_transformed_area(const lv_obj_t * obj, lv_area_t * area, bool re
 {
     lv_point_t p[4] = {
         {area->x1, area->y1},
```

Comment:
I seem to have fixed it with the following code, but I don't know why.
```suggestion
        {area->x1 - 1, area->y1 - 1},
        {area->x1 - 1, area->y2 + 2},
        {area->x2 + 2, area->y1 - 1},
        {area->x2 + 2, area->y2 + 2},
```